.github/workflows/BushelKit.yml

```
@@ -26,7 +29,7 @@ jobs:
         uses: codecov/codecov-action@v4       
         with:
           fail_ci_if_error: true
-          flags: swift-${{ matrix.swift-version }},ubuntu
+          flags: ${{ matrix.image }}
           verbose: true        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)